src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
         ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Bool -> Html Msg
+viewClaimModal loggedIn claim showClaimModal =
+    let
+        t =
+            loggedIn.shared.translators.t
+
+        greenTextTitleStyle =
+            "uppercase text-green text-xs"
+
+        claimVerifiersList =
+            "block mt-6 h-32 space-y-4"
+
+        photoAndTagName =
+            div []
+                [ Profile.viewLarge loggedIn.shared loggedIn.accountName claim.claimer ]
+
+        claimDateAndState =
+            let
+                date datetime =
+                    Just datetime
+                        |> Utils.posixDateTime
+                        |> Strftime.format "%d %b %Y" Time.utc
+
+                ( claimStatusPhrase, claimStatus, textColor ) =
+                    case claim.status of
+                        Approved ->
+                            ( t "claim.title_approved.1", t "claim.approved", "text-2xl font-bold lowercase text-green" )
+
+                        Rejected ->
+                            ( t "claim.title_rejected.1", t "claim.disapproved", "text-2xl font-bold lowercase text-red" )
+
+                        Pending ->
+                            ( t "claim.title_under_review.1", t "claim.pending", "text-2xl font-bold lowercase text-gray-600" )
+
+                claimDate =
+                    let
+                        claimedOn =
+                            t "claim.claimed_on"
+
+                        dateTime =
+                            date claim.createdAt
+                    in
+                    claimedOn ++ " " ++ dateTime
```

Comment:
This is what `shared.translators.tr` is for! Not all languages have this format (what if we decide to support a language that reads right-to-left?), so the `tr` function accepts a string and a list of replacements. Each translation file is responsible for placing these replacements. You can see an example usage [here](https://github.com/cambiatus/frontend/blob/717c820a54df355498f257cb5efa33ffb363f8f3/src/elm/Page/Login.elm#L173)